docs/integrations/code-conventions.md

```diff
@@ -744,7 +744,7 @@ This class is used to return outputs. This object represents an entry in warroom
 | outputs_prefix    | str    | Should be identical to the prefix in the yml contextPath in yml file. for example:         CortexXDR.Incident                                                                              |
 | outputs_key_field | str    | Primary key field in the main object. If the command returns Incidents, and of the properties of Incident is incident_id, then outputs_key_field='incident_id'                             |
 | outputs           | object | The data to be returned and will be set to context                                                                                                                                         |
-| readable_output    | str   | (Optional) markdown string that will be presented in the warroom, should be human readable -  (HumanReadable) - if not set, readable output will be generated via tableToMarkdown function |
+| readable_output    | str   | (Optional) markdown string that will be presented in the War Room, should be human readable -  (HumanReadable) - if not set, readable output will be generated via tableToMarkdown function. To create a War Room entry only, and not write any data to context, use `readable_output` and do _not_ use `outputs`. |
```

Comment:
I don't think we should add this here, this is not related to the "readable_output" argument - 
If one doesn't want to write anything to the context they should just not pass the outputs argument to the commandResults struct.
Did you encounter any problem with the current implementation?

Reply:
I see what you mean. I opened this because I had a client ask about this use case – "is it possible to post a note to the war room without adding content to the context"? – and I thought the documentation could be clearer in describing how to implement this, since the documentation makes it look like the `outputs` arg is required.

Maybe we could add a simple code sample for this particular use case. Might be better on https://xsoar.pan.dev/docs/integrations/context-and-outputs. What do you think?

Reply:
This is simply a bug in the documentation - the "outputs' field is indeed optional. I think this will make things clearer to the customer, WDYT ?

Reply:
Yes agreed, this looks good to me.